<launch>
    <node pkg="libuavf_2024" exec="imaging_node.py" name="imaging_server" />
</launch>